### PR TITLE
Features/improved UI wallet

### PR DESCRIPTION
**Pull request by natgate**

### CODE REVIEW

**Review by jules20022000 — APPROVED** [review]
Good initiative, it will begin the refactoring for the UI.

File: app/src/main/res/layout/walletcard.xml [hunk 1]
```diff
@@ -0,0 +1,386 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (Scott2Williams) [inline app/src/main/res/layout/walletcard.xml:2]: Be wary of the many pixel-defined dimensions for UI components. Some of the larger items like the `CardViews` and layouts could have parent- or content-defined heights and widths.

**Review by Scott2Williams — APPROVED** [review]
I like the new UI components and I understand how large of a task this is. Good work.

File: app/src/main/res/layout/walletcard.xml (hunk 1 above)
Comment (natgate) [reply app/src/main/res/layout/walletcard.xml:2]: Thanks I will take that in note for next time!

**Review by paul-ndl — APPROVED** [review]
nice job! it will make the UI more consistent on the wallet part.

**Pull request merged**